src/scratchtocatrobat/tools/svgtopng.py

```diff
@@ -330,21 +210,18 @@ def _parse_and_rewrite_svg_file(svg_input_path, svg_output_path):
     if 'width' in root.attrib and float((root.attrib['width']).replace('px', '')) <= 0:
         root.attrib['width'] = '1'
 
-    if 'viewBox' not in root.attrib:
-        for child in root:
-            if re.search('.*}g', child.tag) != None:
-                if 'transform' in child.attrib:
-                    matrix_transform_attrib = child.attrib['transform']
-                    matrix_transform_attrib = re.sub(r"matrix\((\s?-?[0-9]+(\.[0-9]*)?,){4}", "matrix(1, 0, 0, 1,", matrix_transform_attrib)
-                    child.attrib['transform'] = matrix_transform_attrib
-                break
-    if 'viewBox' in root.attrib:
-        del root.attrib['viewBox']
-
-    for child in root:
+
```

Comment:
we only iterated over elements directly in root, now we iterate over all

also i removed the del root.attrib['viewBox'] because I don't think this is needed